### PR TITLE
[Merged by Bors] - Enable logging

### PR DESCRIPTION
**Pull request by maltesander**
# Description

- enabled logging via logback

## Question

Collect logs from `create-reportin-task` (Job)?

fixes: https://github.com/stackabletech/nifi-operator/issues/415

## Review Checklist

- [x] Code contains useful comments
- [x] CRD change approved (or not applicable)
- [x] (Integration-)Test cases added (or not applicable)
- [x] Documentation added (or not applicable)
- [x] Changelog updated (or not applicable)
- [x] Cargo.toml only contains references to git tags (not specific commits or branches)
- [x] Helm chart can be installed and deployed operator works (or not applicable)

Once the review is done, comment `bors r+` (or `bors merge`) to merge. [Further information](https://bors.tech/documentation/getting-started/#reviewing-pull-requests)

### CODE REVIEW

**Comment by maltesander** [comment]
test: https://ci.stackable.tech/view/02%20Operator%20Tests%20(custom)/job/nifi-operator-it-custom/22/

File: tests/templates/kuttl/upgrade/02-install-nifi.yaml.j2 [hunk 1]
```diff
@@ -29,11 +29,15 @@ spec:
           adminCredentialsSecret: nifi-admin-credentials-simple
     sensitiveProperties:
       keySecret: nifi-sensitive-property-key
+{% if lookup('env', 'VECTOR_AGGREGATOR') %}
+    vectorAggregatorConfigMapName: nifi-vector-aggregator-discovery
```
Comment (siegfriedweber) [inline tests/templates/kuttl/upgrade/02-install-nifi.yaml.j2:33]:
```suggestion
    vectorAggregatorConfigMapName: vector-aggregator-discovery
```

File: tests/templates/kuttl/upgrade/05-upgrade-nifi.yaml.j2 [hunk 2]
```diff
@@ -14,11 +14,15 @@ spec:
           adminCredentialsSecret: nifi-admin-credentials-simple
     sensitiveProperties:
       keySecret: nifi-sensitive-property-key
+{% if lookup('env', 'VECTOR_AGGREGATOR') %}
+    vectorAggregatorConfigMapName: nifi-vector-aggregator-discovery
```
Comment (siegfriedweber) [inline tests/templates/kuttl/upgrade/05-upgrade-nifi.yaml.j2:18]:
```suggestion
    vectorAggregatorConfigMapName: vector-aggregator-discovery
```

**Review by siegfriedweber — CHANGES_REQUESTED** [review]
On the console, all logs are wrapped into another log entry:

```
2023-01-30 12:57:45,332 INFO [NiFi logging handler] org.apache.nifi.StdOut 2023-01-30 12:57:45,332 INFO [pool-9-thread-1] o.a.n.c.r.WriteAheadFlowFileRepository Initiating checkpoint of FlowFile Repository
```

In the log file, the console output is replicated:

```xml
<log4j:event logger="org.apache.nifi.controller.repository.WriteAheadFlowFileRepository"
             timestamp="1675083205329" level="INFO" thread="pool-9-thread-1">
  <log4j:message>Initiating checkpoint of FlowFile Repository</log4j:message>

</log4j:event>

<log4j:event logger="org.apache.nifi.StdOut"
             timestamp="1675083205329" level="INFO" thread="NiFi logging handler">
  <log4j:message>2023-01-30 12:53:25,329 INFO [pool-9-thread-1] o.a.n.c.r.WriteAheadFlowFileRepository Initiating checkpoint of FlowFile Repository</log4j:message>

</log4j:event>
```

**Comment by maltesander** [comment]
Last test hopefully: https://ci.stackable.tech/view/02%20Operator%20Tests%20(custom)/job/nifi-operator-it-custom/23/

**Review by siegfriedweber — APPROVED** [review]
LGTM

**Pull request closed without merging**